fw.json

```
@@ -21,6 +21,8 @@
         ["FrSky X-Lite", "xlite-"],
         ["FrSky X-Lite S", "xlites-"],
         ["HelloRadioSky V16", "v16-"],
+        ["HelloRadioSky V14", "v14-"],
+        ["HelloRadioSky V12", "v12-"],
```

Alphabetical order! ;)
```suggestion
        ["HelloRadioSky V12", "v12-"],
        ["HelloRadioSky V14", "v14-"],
        ["HelloRadioSky V16", "v16-"],
```